Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
```

Comment:
### [standard](https://github.com/testdouble/standard)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.9.1 → 1.16.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/standard/1.9.1/1.16.1)_]





<details>
<summary>Commits</summary>

A change of **45** commits. See the full changes on [the compare page](https://github.com/testdouble/standard/compare/v1.9.1...v1.16.1).

These are the first 10 commits:
- (bd941e2) [[ 2022-04-11 ] - Update dependencies](https://github.com/testdouble/standard/commit/bd941e2332b9e3bb86359fd362ebe30020cc7946)
- (bff22fa) [Configures the new cops in 1.27.0](https://github.com/testdouble/standard/commit/bff22fa6e08c732a4f975597fc4c4aa373b37f94)
- (59a4439) [Merge pull request #406 from testdouble/update-deps-2022-04-…](https://github.com/testdouble/standard/commit/59a4439bbc2e07e3ca7e226d6cec96e95de32a10)
- (327f177) [🚗 1.10.0](https://github.com/testdouble/standard/commit/327f17782413f537d1c76034a5ef7415b52d1bf4)
- (1ccf1b7) [[ 2022-04-25 ] - Update dependencies](https://github.com/testdouble/standard/commit/1ccf1b731c8d7f0a222ecf681e63ad07cb5cac54)
- (71ac2fe) [Corrects IgnoredPatterns problem in Layout/IndentationWidth](https://github.com/testdouble/standard/commit/71ac2fe5cf75fa17a41c8b486a1d1633af56322c)
- (ef439a1) [Configures new cops.](https://github.com/testdouble/standard/commit/ef439a1da5dc253b208472bc48d633743c3bdbbd)
- (0601ce6) [Merge pull request #409 from testdouble/update-deps-2022-04-…](https://github.com/testdouble/standard/commit/0601ce694a1d2fa597ab84db44b30aed79ad8701)
- (e6130ac) [🛣 v1.11.0](https://github.com/testdouble/standard/commit/e6130acce8d336a7083813cc216bdddcafa60155)
- (1ec52c6) [enforce :"foo-bar" over :\\foo-bar\ to be consistent with ot…](https://github.com/testdouble/standard/commit/1ec52c6f21de0e4b481d6914bfb6e73a083de28a)


</details>